haplotypecaller-gvcf-gatk4.wdl

```diff
@@ -208,8 +215,15 @@ task HaplotypeCaller {
       -I ~{input_bam} \
       -L ~{interval_list} \
       -O ~{output_filename} \
-      -contamination ~{default=0 contamination} ~{true="-ERC GVCF" false="" make_gvcf} \
-      -G StandardAnnotation -G AS_StandardAnnotation -G StandardHCAnnotation
+      -contamination ~{default=0 contamination} \
+      -G StandardAnnotation -G StandardHCAnnotation ~{true="-G AS_StandardAnnotation" false="" make_gvcf} \
+      -new-qual \
```

Comment:
Is this arg still in 4.1.7?  I was pretty sure we took it out a few versions after it became the default.

Reply:
`AS_StandardAnnotation` yes. link to the walker file in master [here](https://github.com/broadinstitute/gatk/blob/master/src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_StandardAnnotation.java). And i double checked with the 4.1.7.1 and HC threw an error when i misspelled AS_StandardAnnotation but didn't error when i spelled it correctly.

Reply:
It's the `-new-qual` I was wondering about, specifically.  But if it runs, then that's fine with me.

Reply:
Ahh, your right `new-qual` has been removed from the latest version and it does not run if inserted in the command. I'll remove it